server/items.csv

```diff
@@ -0,0 +1,31 @@
+Garlic Edamame,400,Appetizer
```

Comment:
I took the price and name straight from the spreadsheet and pdf of the menu attached to the story on trello. I made up the categories.